src/dotnet-install.ps1

```diff
@@ -69,6 +69,8 @@
 .PARAMETER ProxyUseDefaultCredentials
     Default: false
     Use default credentials, when using proxy address.
+.PARAMETER ProxyBypassList
+    If set with ProxyAddress, will provide the list of comma separated urls that will bypass the proxy
```

Comment:
Can we update this to reflect the latest changes? Instead of comma separated urls, we have a string array now.

Reply:
Imho this is correct for both cases "abc", "abc" "abc, abc" , both could be considered comma separated list, but just the first is an array of strings.